Packs/CarbonBlackDefense/Integrations/CarbonBlackDefenseV2/CarbonBlackDefenseV2.py

```diff
@@ -0,0 +1,1169 @@
+    policy_id = args.get('policyId')
+    rule_id = args.get('ruleId')
+
+    if not policy_id or not rule_id:
+        return "Missing required arguments."
+    res = client.delete_rule_from_policy(int(policy_id), int(rule_id))
+    readable_output = tableToMarkdown("Carbon Black Defense Delete Rule From Policy",
+                                      res,
+                                      headerTransform=string_to_table_header)
+
+    return CommandResults(
+        readable_output=readable_output,
+        raw_response=res
+    )
+
+
+def find_events_command(client: Client, args: dict):
+    alert_category = argToList(args.get('alert_category'))
+    blocked_hash = argToList(args.get('blocked_hash'))
+    device_external_ip = argToList(args.get('device_external_ip'))
+    device_id = argToList(args.get('device_id'))
+    device_internal_ip = argToList(args.get('device_internal_ip'))
+    device_name = argToList(args.get('device_name'))
+    device_os = argToList(args.get('device_os'))
+    event_type = argToList(args.get('event_type'))
+    parent_hash = argToList(args.get('parent_hash'))
+    parent_name = argToList(args.get('parent_name'))
+    parent_reputation = argToList(args.get('parent_reputation'))
+    process_cmdline = argToList(args.get('process_cmdline'))
+    process_guid = argToList(args.get('process_guid'))
+    process_hash = argToList(args.get('process_hash'))
+    process_name = argToList(args.get('process_name'))
+    process_pid = argToList(args.get('process_pid'))
+    process_reputation = argToList(args.get('process_reputation'))
+    process_start_time = argToList(args.get('process_start_time'))
+    process_terminated = argToList(args.get('process_terminated'))
+    process_username = argToList(args.get('process_username'))
+    sensor_action = argToList(args.get('sensor_action'))
+    query = args.get('query')
+    rows = arg_to_number(args.get('rows'))
+    start = arg_to_number(args.get('start'))
+    time_range = args.get('timerange')
+
+    if time_range:
+        time_range = json.loads(time_range)
+
+    res = client.get_events(alert_category, blocked_hash, device_external_ip, device_id, device_internal_ip,
+                            device_name, device_os, event_type, parent_hash, parent_name,
+                            parent_reputation, process_cmdline, process_guid, process_hash, process_name,
+                            process_pid, process_reputation, process_start_time, process_terminated,
+                            process_username, sensor_action, query, start, time_range, rows)
+
+    if type(res) is str:
+        return res
+
+    readable_output = tableToMarkdown('Carbon Black Defense Events Search',
+                                      res,
+                                      headerTransform=string_to_table_header)
+
+    return CommandResults(
+        outputs_prefix='CarbonBlackDefense.Events.Search',
+        outputs_key_field='job_id',
```

Comment:
Lets talk about this command's outputs once again. Seems to me like we can get duplicate incidents